doc/reference/storage_ceph.md

```diff
@@ -1,56 +1,110 @@
+
+Ceph provides different components for block storage and for file systems.
+<!-- Include end Ceph intro -->
+
+Ceph {abbr}`RBD (RADOS Block Device)` is Ceph's block storage component that distributes data and workload across the Ceph cluster.
+It uses thin provisioning, which means that it is possible to over-commit resources.
+
+## Terminology
+
+<!-- Include start Ceph terminology -->
+Ceph uses the term *object* for the data that it stores.
+The daemon that is responsible for storing and managing data is the *Ceph {abbr}`OSD (Object Storage Daemon)`*.
+Ceph's storage is divided into *pools*, which are logical partitions for storing objects.
+They are also referred to as *data pools*, *storage pools* or *OSD pools*.
+<!-- Include end Ceph terminology -->
+
+Ceph block devices are also called *RBD images*, and you can create *snapshots* and *clones* of these RBD images.
+
+## `ceph` driver in LXD
+
+```{note}
+To use the Ceph RBD driver, you must specify it as `ceph`.
+This is slightly misleading, because it provides only Ceph RBD (block storage) functionality, not full Ceph functionality.
+Therefore, the `ceph` driver can only be used for storage volumes with content type `block`.
+
+For storage volumes with content type `filesystem`, use the {ref}`storage-cephfs` driver.
+```
+<!-- Include start Ceph driver cluster -->
+Unlike other storage drivers, this driver does not set up the storage system but assumes that you already have a Ceph cluster installed.
+<!-- Include end Ceph driver cluster -->
+
+<!-- Include start Ceph driver remote -->
+This driver also behaves differently than other drivers in that it provides remote storage.
+As a result and depending on the internal network, storage access might be a bit slower than for local storage.
+On the other hand, using remote storage has big advantages in a cluster setup, because all cluster members have access to the same storage pools with the exact same contents, without the need to synchronize storage pools.
+<!-- Include end Ceph driver remote -->
+
+The `ceph` driver in LXD uses RBD images for images, and snapshots and clones to create instances and snapshots.
+
+<!-- Include start Ceph driver control -->
+LXD assumes that it has full control over the OSD storage pool.
+Therefore, you should never maintain any file system entities that are not owned by LXD in a LXD OSD storage pool, because LXD might delete them.
+<!-- Include end Ceph driver control -->
+
+Due to the way copy-on-write works in Ceph RBD, parent RBD images can't be removed until all children are gone.
+As a result, LXD automatically renames any objects that are removed but still referenced.
+Such objects are kept with a  `zombie_` prefix until all references are gone and the object can safely be removed.
+
+### Limitations
+
+The `ceph` driver has the following limitations:
+
+Sharing the OSD storage pool between instances
```

Comment:
We need to be careful around the term "instances" here, as everywhere else in the docs an "instance" is either a container or a VM, but here we are talking about a LXD installation (I think, given the line below). So perhaps "LXD installation" rather than "LXD instance" would be clearer.